static/js/modules/columns.js

```diff
@@ -29,6 +29,19 @@ var amendmentIndicatorColumn = {
   },
 };
 
+var versionColumn = {
+  data: 'most_recent',
+  className: 'hide-panel hide-efiling column--med min-desktop',
+  render: function(data) {
+    if (data === true) {
+      return '<div class="icon-circle--check-outline--inline--left"></div>Most recent version';
```

Comment:
Small thing, but I see that elsewhere we usually do `<i>` for icons instead of `<div>`